environment/src/main/java/com/sequenceiq/environment/environment/v1/converter/EnvironmentApiConverter.java

```diff
@@ -234,7 +236,8 @@ private AwsParametersDto awsParamsToAwsParameters(AwsEnvironmentParameters aws,
     private AwsDiskEncryptionParametersDto awsDiskEncryptionParametersToAwsDiskEncryptionParametersDto(
             AwsDiskEncryptionParameters awsDiskEncryptionParameters) {
         AwsDiskEncryptionParametersDto.Builder awsDiskEncryptionParametersDto = AwsDiskEncryptionParametersDto.builder()
-                .withEncryptionKeyArn(awsDiskEncryptionParameters.getEncryptionKeyArn());
+                .withEncryptionKeyArn(Optional.ofNullable(awsDiskEncryptionParameters.getEncryptionKeyArn())
+                        .orElse(null));
```

Comment:
So, as per my [previous comment](https://github.com/hortonworks/cloudbreak/pull/11874#discussion_r768039630), `awsDiskEncryptionParameters` (which was obtained from `request.getAwsDiskEncryptionParameters()`) may be `null`, so this one is still prone to NPE. The proper approach (if using `Optional`) would be something like:

```
.withEncryptionKeyArn(Optional.ofNullable(awsDiskEncryptionParameters).map(AwsDiskEncryptionParameters::getEncryptionKeyArn).orElse(null))
```